packages/backend/src/database/entities/savedCharts.ts

```diff
@@ -251,3 +257,27 @@ export type DBFilteredAdditionalMetrics = Pick<
             | 'format_options'
         >
     >;
+
+export const SavedChartCustomSqlDimensionsTableName =
+    'saved_queries_version_custom_sql_dimensions';
+
+export type DbSavedChartCustomSqlDimension = {
+    saved_queries_version_custom_sql_dimensions_uuid: number;
+    saved_queries_version_id: number;
+    id: string;
+    name: string;
+    table: string;
```

Comment:
To add it under the correct table in the sidebar

Reply:
It might be defined across many tables right?

Reply:
Not sure what you mean. 
We can define custom metrics under any of the explore tables.

Reply:
When you click +add dimension, it could reference any of the tables right, so it doesn't live on one table alone.

e.g. `${table1.dim1} + ${table2.dim2}`

Reply:
correct, we will support them referencing for any table from that explore ( not included in this PR ). This value is only used to place the custom dimension in the UI and not for SQL generation.

Reply:
Can we remove it and have it live with custom metrics "outside" of the tables?

Reply:
🌴 oh dear. ok let's just make it extremely clear that it's only for the UI. I'm worried that devs are going to get misled by it